test/test.js

```diff
@@ -76,6 +76,19 @@ describe('cacheman-file', function() {
     });
   });
 
+  it('should sanitize key', function(done) {
```

Comment:
Sounds totally reasonable (and you're right, the tests should know). Let me rework the test.
Thanks!